app/models/configuration_script.rb

```diff
@@ -1,3 +1,17 @@
 class ConfigurationScript < ActiveRecord::Base
-  belongs_to :configuration_manager, :class_name => 'ManageIQ::Providers::ConfigurationManager'
+  belongs_to  :manager,
+              :class_name  => 'ManageIQ::Providers::ConfigurationManager',
+              :foreign_key => :configuration_manager_id
+  include ProviderObjectMixin
+
+  def run(vars=nil)
+    extra_vars = vars.reverse_merge(variables)
```

Comment:
@syncrou 
What is the data format of the variables, is it a Hash, JSON, YAML?
If the JobTemplate has the extra variables defined in it, it can be either in YAML or JSON?
If the Job Template has the extra variables defined in YAML, we get a failure in 
ansible_tower_client-e9317bb0b23e/lib/ansible_tower_client/job_template.rb:16

``AnsibleTowerClient::InvalidJson: Error on JSON::ParserError: #<JSON::ParserError: 757: unexpected token at '---
  host: ManageIQ(TinaF)
  user: root
  pkg: abrt
  sleep: 3'>